src/Services/NodeUptimeNotificationService.ts

```diff
@@ -103,9 +103,10 @@ export class NodeUptimeNotificationService {
                 logger.info(`Sending mail to: ${user.email} for node: ${node.id}::${node.address}`);
                 if (config.env != "dev") {
                     logger.info(`Going to send email to ${user.id}.`);
+                    const nodeName = node.name ? node.name : node.id;
```

Comment:
Not only id but `Node ${node.id}` as without context it looks weird.

Also, you can use syntax `node.name || "Node ${node.id}"` :)